service/controller/key/cluster.go

```diff
@@ -63,6 +63,10 @@ const (
 	StackTCNPF = "tcnpf"
 )
 
+const (
+	LifeCycleHookNodePool = "NodePool"
```

Comment:
Defining this for Node Pools now. Later we add it for the Control Plane as well.